elasticdl/python/common/k8s_client.py

```diff
@@ -67,14 +68,23 @@ def __init__(
                 Used as pod name prefix and value for "elasticdl" label.
             event_callback: If not None, an event watcher will be created and
                 events passed to the callback.
+            force_use_kube_config_file: If true, force load the cluster config
+                from ~/.kube/config file. Otherwise, if it's in a process
+                running in a K8S environment, it loads the incluster config,
+                if not, load the kube config file.
```

Comment:
"force load the cluster config" -> "force to load the cluster config"

"load the kube config file."  -> "loads the kube config file."

Reply:
Good suggestion, updated.